tests/test_instrument.py

```diff
@@ -33,10 +33,21 @@
 
 class DummyCam(Instrument):
 
+    class FilterDefinitions:
+        """Stopgap until Instrument is moved into obs_base and we can use the
+        real FilterDefinitions there.
+        """
+        def defineFilters(self):
+            pass
+    filterDefinitions = FilterDefinitions()
```

Comment:
It looks like the `register` implementation below doesn't call `_registerFilters` or otherwise use `FilterDefinitions`.  Do you need this shim?